binding.gyp

```diff
@@ -1,39 +1,27 @@
 {
+  'variables':{
+    'INTEL_OPENVINO_DIR' : '<!@(node -p \"process.env.INTEL_OPENVINO_DIR\")>',
```

Comment:
This requires to setup openvino environment variables beforehand. Could you please update the build instructions in README.md accordingly?

Reply:
Fixed in latest commits.